pkgs/development/libraries/unixODBCDrivers/default.nix

```diff
@@ -67,7 +67,7 @@
     };
 
     nativeBuildInputs = [ cmake ];
-    buildInputs = [ unixODBC mysql55 ];
+    buildInputs = [ unixODBC mysql ];
```

Comment:
these derivations are marked as broken, and seem to fetch mysql 5.3 components even! Can we get these to work with a later mysql version, or maybe drop them too?

Reply:
It would be a real shame to drop instead of fix...

Reply:
Maybe this will be fixed in a follow up PR.